docs/3.0rc/cloud/self-host.mdx

```diff
@@ -46,12 +82,13 @@ Go to your terminal session and run this command to set the API URL to point to
 prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
 ```
 
-!!! tip "`PREFECT_API_URL` required when running Prefect inside a container"
-    You must set the API server address to use Prefect within a container, such as a Docker container.
+<Info>
+You must set the API server address--`PREFECT_API_URL`--to use Prefect within a container, such as a Docker container.
```

Comment:
```suggestion
You must set the API server address—`PREFECT_API_URL`—to use Prefect within a container, such as a Docker container.
```

Noting that our use of em dash style variations is a thing. Let's standardize on [Google style](https://developers.google.com/style/dashes#:~:text=After%20the%20Compose%20key%20is,Then%20press%20Return%20.). Their rec is:

> Em dashes
To indicate a break in the flow of a sentence—or an interruption—use an em dash, also known as a long dash. Don't put a space before or after it.

You can type the em dash character in various ways:

HTML
&mdash;

macOS
Press Option+Shift+hyphen.